src/Integrations/Laravel/BackupManagerServiceProviderLaravel5.php

```diff
@@ -0,0 +1,166 @@
+<?php namespace BigName\BackupManager\Integrations\Laravel;
+
+use BigName\BackupManager\Databases;
+use BigName\BackupManager\Filesystems;
+use BigName\BackupManager\Compressors;
+use Symfony\Component\Process\Process;
+use Illuminate\Support\ServiceProvider;
+use BigName\BackupManager\Config\Config;
+use BigName\BackupManager\ShellProcessing\ShellProcessor;
+
+/**
+ * Class BackupManagerServiceProvider
+ * @package BigName\BackupManager\Integrations\Laravel
+ */
+class BackupManagerServiceProviderLaravel5 extends ServiceProvider
+{
+    protected $defer = true;
+
+    /**
+     * Bootstrap the application events.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        $configPath = __DIR__ . '/../../../config/';
+        $this->publishes([
+            $configPath => config_path('backup-manager/'),
+        ], 'backup-manager');
+    }
+
+    /**
+     * Register the service provider.
+     *
+     * @return void
+     */
+    public function register()
+    {
+
+        $this->mergeConfigFrom(config_path('backup-manager/storage.php'), 'backup-manager::storage');
```

Comment:
Hi @GrahamCampbell ,

I am not sure I understand, can you tell me which line you mean specifically?  In Taylor's [docs](http://laravel.com/docs/5.0/packages#configuration) that "To publish a configuration file, just use the publishes method from the boot method of your service provider:"  and "To merge the configurations, use the mergeConfigFrom method within your service provider's register method".  Can you please advise further what the code should be?  I tested it and it seems to be working.